manual/manual/cmds/comp.etex

```diff
@@ -100,6 +100,18 @@ phase can be run directly, as in:
 The produced file has the executable bit set, and it manages to launch
 the bytecode interpreter by itself.
 
+The compiler is able to emit some information on its internal stages.
+Upon successful completion of the type checking procedure, type information is
+used to create an typed abstract syntax tree (AST). It contains complete
+information about the location and the specific type of each term in the
+source file. In case of unsuccessful type checking, a partial typed tree
```

Comment:
There might be some tension here because you first tell the reader that the typed AST is only produced after a sucessfull type checking phase, then adds a correction telling that a (partial) AST is produced otherwise. This opposition seems unnecessary. You could simply write that a typed AST is produced during type checking. Then you could speak about the fact that typed AST is partial if typechecking was unsuccessfull (if you wish).